src/data_provider/tests/sysInfoRpmPackageManager/main.cpp

```diff
@@ -0,0 +1,18 @@
+/*
+ * Wazuh SysInfo
+ * Copyright (C) 2015-2021, Wazuh Inc.
+ * March 16, 2021.
```

Comment:
Hi Team

This file is new, the date must be this month or last month.

Regards
Hanes